docdl/plugins/handyvertrag.py

```diff
@@ -0,0 +1,122 @@
+        current_url = self.webdriver.current_url
+        # find entry field
+        password = self.webdriver.find_element(
+            By.XPATH,
+            "//input[@id='UserLoginType_password']"
+        )
+        # wait for entry field
+        WebDriverWait(self.webdriver, self.TIMEOUT).until(
+            EC.visibility_of(password)
+        )
+        # send password
+        password.send_keys(self.password)
+        # submit form
+        password.submit()
+        # wait for page to load
+        current_url = self.wait_for_urlchange(current_url)
+        # wait for either login success, failure or "accept" button
+        WebDriverWait(self.webdriver, self.TIMEOUT).until(
+            EC.presence_of_element_located(
+                (By.XPATH,
+                    "//img[contains(@alt, 'LOGOUT')] | "
+                    "//div[contains(@data-test-id, 'unified-login-error')]")
+            )
+        )
+        # click "confirm" button for cookies if there is one
+        cookiebutton = self.webdriver.find_element(
+                By.XPATH, '//*[@id="consent_wall_optin"]')
+        if cookiebutton:
+            cookiebutton.click()
+
+        # Login failed
+        return self.webdriver.find_elements(
+            By.XPATH, "//img[contains(@alt, 'LOGOUT')]"
+        )
+
+    def logout(self):
+        # logoutbutton = self.webdriver.find_element(
+        #       By.XPATH, '//span[@id, "logoutLink"]/a')
+        # logoutbutton.click()
+        self.webdirver.get(self.URL_LOGOUT)
+
+    def documents(self):
+        """fetch invoices"""
+        self.webdriver.get(self.URL_INVOICES)
+
+        invoice_collapser = self.webdriver.find_elements(
+                By.XPATH, '//*[contains(@id, "heading-rechnungen-")]/button')
+
+        for row in invoice_collapser:
+            row.click()
+            description = row.text
+            date = description.split()[-1]
+
+            # type invoice
+            link = row.parent.find_elements(By.LINK_TEXT, 'Rechnung')
+            if len(link) == 1:
+                url = link[0].get_attribute("href")
+                idn = url.split('/')[-1]
+                yield docdl.Document(
+                    url=url,
+                    attributes={
+                        'id': idn,
+                        'date': docdl.util.parse_date(date),
+                        'category': 'invoice',
+                        'subject': description,
+                    })
+
+            # type connection report
+            link = row.parent.find_elements(
+                    By.LINK_TEXT, 'Einzelverbindungsnachweis')
+            if len(link) == 1:
+                url = link[0].get_attribute("href")
+                idn = url.split('/')[-1]
+                yield docdl.Document(
+                    url=url,
+                    attributes={
+                        'id': idn,
+                        'date': docdl.util.parse_date(date),
+                        'category': 'connection_report',
```

Comment:
i think "call log" would be the correct translation, but i'm not sure. The category can be plugin specific but I'd suggest trying to keep it consistent with documentation.